python/dolfinx/wrappers/fem.cpp

```diff
@@ -138,6 +139,25 @@ void declare_functions(py::module& m)
       py::arg("b"), py::arg("L"), py::arg("constants"), py::arg("coeffs"),
       "Assemble linear form into an existing vector with pre-packed constants "
       "and coefficients");
+  // Matrix
+  m.def(
+      "assemble_matrix",
+      [](dolfinx::la::MatrixCSR<T>& A, const dolfinx::fem::Form<T>& form,
+         const std::vector<std::shared_ptr<const dolfinx::fem::DirichletBC<T>>>&
+             bcs)
+      { dolfinx::fem::assemble_matrix<T>(A.mat_add_values(), form, bcs); },
+      "Experimental.");
+  // NOTE: This is relative slow. Need to investigate reasons
```

Comment:
remove commented code?

Reply:
I'd like to leave it in for the time being - might come back to it later.